### PR TITLE
Deployment alignments

### PR DESCRIPTION
**Pull request by SteveMattar**
- refactor: globalnet should be under the broker
- feat: support ovn
- feat: add lighthouse to e2e matrix

### CODE REVIEW

File: submariner/questions.yml [hunk 1]
```diff
@@ -136,3 +136,9 @@ questions:
   group: "Advanced Configuration"
   description: "Cable driver implementation"
   label: "Cable Driver"
+- variable: submariner.serviceDiscovery
+  type: boolean
+  default: false
+  group: "Advanced Configuration"
+  description: "Enable Multi Cluster Service Discovery"
```
Comment (dfarrell07) [inline submariner/questions.yml]: Maybe "Enable multicluster service discovery"?

File: submariner-operator/questions.yml [hunk 2]
```diff
@@ -92,13 +92,26 @@ questions:
...
-  description: "Enter the globalnet CIDR (i.e. 169.254.1.0/24) for this cluster if using globalnet"
-  type: string
-  label: "Globalnet CIDR"
-  group: "CIDR Configuration"
-  required: false
+- variable: submariner.serviceDiscovery
+  type: boolean
+  default: false
+  group: "Advanced Configuration"
+  description: "Enable Multi Cluster Service Discovery"
```
Comment (dfarrell07) [inline submariner-operator/questions.yml]: Maybe "Enable multicluster service discovery"?

**Review by dfarrell07 — APPROVED** [review]
Looks great, thanks @SteveMattar!

**Comment by dfarrell07** [comment]
I retriggered your CI and everything is passing now @SteveMattar. :partying_face:

**Review by dfarrell07 — CHANGES_REQUESTED** [review]
<ignore, sorry>

**Review by dfarrell07 — APPROVED** [review]

**Review by tpantelis — APPROVED** [review]

**Pull request merged**